CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ option(JSONTOOLKIT_JSONSCHEMA "Build the JSON Toolkit JSON Schema library" ON)
 option(JSONTOOLKIT_JSONPOINTER "Build the JSON Toolkit JSON Pointer library" ON)
 option(JSONTOOLKIT_JSONL "Build the JSON Toolkit JSONL library" ON)
 option(JSONTOOLKIT_TESTS "Build the JSON Toolkit tests" OFF)
+option(JSONTOOLKIT_BENCHMARKS "Build the JSON Toolkit benchmarks" OFF)
```

Comment:
```suggestion
option(JSONTOOLKIT_BENCHMARK "Build the JSON Toolkit benchmark" OFF)
```

Reply:
Now I wonder, why do we have plural for test?